x-pack/plugins/security_solution/public/cloud_security_posture/components/index.tsx

```diff
@@ -25,10 +26,27 @@ export const EntityInsight = <T,>({
   const { euiTheme } = useEuiTheme();
   const getSetupStatus = useCspSetupStatusApi();
   const hasMisconfigurationFindings = getSetupStatus.data?.hasMisconfigurationsFindings;
+  const hasVulnerabilitiesFindings = getSetupStatus.data?.hasVulnerabilitiesFindings;
+  const insightContent: React.ReactElement[] = [];
 
+  if (hasMisconfigurationFindings)
+    insightContent.push(
+      <>
+        <MisconfigurationsPreview name={name} fieldName={fieldName} isPreviewMode={isPreviewMode} />
+        <EuiSpacer size="m" />
+      </>
+    );
+  if (hasVulnerabilitiesFindings && fieldName === 'host.name')
```

Comment:
the condition is duplicated on line 39 and 49, I would move it to a single constant to avoid them keeping out of sync